src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -124,7 +123,7 @@ const FeedModalItem = (props: FeedEventProps) => {
                   { flexDirection: 'row', justifyContent: 'center', alignItems: 'center', marginRight: 15 },
                 ]}
               >
-                {/* {!eventSettings.withoutAmount && ownerAddress.length > 0 && walletChatEnabled && (
+                {!eventSettings.withoutAmount && ownerAddress.length > 0 && (
```

Comment:
There is a PR open but if you apply the other fixes
https://github.com/GoodDollar/GoodDAPP/pull/4086/files

please add this change as well (the fix for not having the chat showing up on the wrong transactions like bridging/claiming etc)

Reply:
why walletChatEnabled flag check was removed ?

Reply:
Thanks for the review, I have updated the PR to include the checks and changes from the mentioned PR